package.json

```diff
@@ -8,6 +8,9 @@
   ],
   "version": "15.2.3",
   "homepage": "http://espadrine.github.com/localeval/",
+  "browser": {
```

Comment:
I can't find the documentation for `"browser"` in [npm's `package.json` reference](https://docs.npmjs.com/files/package.json). Could you explain it please?